_data/afroz23/afroz23.md

```diff
@@ -0,0 +1,7 @@
+---
+name: AFROZ QURAISHI
+institution: Atal Bihari Vajpayee Indian Institute of Information Technology and Management,Gwalior 🚩 
```

Comment:
character count exceeds the given limit, please modify this

Reply:
Modified sir